tests/setup_tests.py

```diff
@@ -11,7 +11,10 @@
 sys.path.append(ROOT_FOLDER)
 
 import CHAID
-
+import CHAID.chaid_vector as CHAIDVector
```

Comment:
This seems wrong. I think the __init__.py should change rather than how you import stuff